content/post/release-of-1.1.0.md

```diff
@@ -0,0 +1,99 @@
+---
+date: "2017-03-09T20:15:00+01:00"
+author: "tboerger"
+title: "Release of 1.1.0"
+tags: ["release"]
+draft: false
+---
+
+We proudly present the release of Gitea version 1.1.0. We have closed [126](https://github.com/go-gitea/gitea/issues?q=is%3Aissue+is%3Aclosed+milestone%3A1.1.0) issues and we also merged [348](https://github.com/go-gitea/gitea/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20milestone%3A1.1.0) pull requests to release this version of Gitea. You can download one of our pre-built binaries from our [downloads page](https://dl.gitea.io/gitea/1.1.0/), you just need to select the correct platform. For further details of the installation follow our [installation guide](https://docs.gitea.io/en-us/install-from-binary/).
+
+<!--more-->
```

Comment:
You could highlight some of the more significant features and include screenshots.